ref/provided-images.md

```diff
@@ -1,36 +1,32 @@
 
-## Official images
-
-The following table lists all images that are available on the official image server, along with their corresponding Android and Ubuntu versions. Please note the support status for each of the images.
-
-| Name                        | Android Version | Ubuntu Version | Support Status     | Available since |
-|-----------------------------|-----------------|----------------|------------|---------------|
-| `jammy:android13:amd64`     | 13              | 22.04          | supported | 1.16 |
-| `jammy:android13:arm64`     | 13              | 22.04          | supported | 1.16 |
-| `jammy:android12:amd64`     | 12              | 22.04          | supported | 1.14 |
-| `jammy:android12:arm64`     | 12              | 22.04          | supported | 1.14 |
-| `jammy:android11:amd64`     | 11              | 22.04          | supported | 1.14 |
-| `jammy:android11:arm64`     | 11              | 22.04          | supported | 1.14 |
-| `jammy:android10:amd64`     | 10              | 22.04          | supported | 1.14 |
-| `jammy:android10:arm64`     | 10              | 22.04          | supported | 1.14 |
-| `bionic:android12:amd64`    | 12              | 18.04          | deprecated as of 1.16 | 1.12 |
-| `bionic:android12:arm64`    | 12              | 18.04          | deprecated as of 1.16 | 1.12 |
-| `bionic:android11:amd64`    | 11              | 18.04          | deprecated as of 1.16 | 1.10 |
-| `bionic:android11:arm64`    | 11              | 18.04          | deprecated as of 1.16 | 1.10 |
-| `bionic:android10:amd64`    | 10              | 18.04          | deprecated as of 1.16 | 1.0  |
-| `bionic:android10:arm64`    | 10              | 18.04          | deprecated as of 1.16  | 1.0  |
-| `bionic:android7:amd64`     | 7               | 18.04          | unsupported as of 1.10 | 1.0 |
-| `bionic:android7:arm64`     | 7               | 18.04          | unsupported as of 1.10 | 1.0 |
+## Supported images
+
+The following table lists supported images available on the official image server, along with their corresponding Android and Ubuntu versions.
+
+| Name                        | Android Version | Ubuntu Version | Available since |
+|-----------------------------|-----------------|----------------|---------------|
+| `jammy:android13:amd64`     | 13              | 22.04          | 1.16 |
+| `jammy:android13:arm64`     | 13              | 22.04          | 1.16 |
+| `jammy:android12:amd64`     | 12              | 22.04          | 1.14 |
+| `jammy:android12:arm64`     | 12              | 22.04          | 1.14 |
+
+## Deprecated images
+The following table lists the images that are available on the official image server but are deprecated. They will become unsupported after two releases.
+
+| Name                        | Android Version | Ubuntu Version | Available since | Deprecated since |
+|-----------------------------|-----------------|----------------|-----------------|--------------------|
```

Comment:
Should we add a column saying with which version (1.21) they get dropped?

Reply:
> Should we add a column saying with which version (1.21) they get dropped?

This is a commitment that we make in advance, so we can add it if we are absolutely sure that it will be dropped in 1.21.

Reply:
That's fair